src/ukify/ukify.py

```diff
@@ -1100,27 +1199,45 @@ def config_example():
             yield f'{key} = {value}'
 
 
-def create_parser():
+def add_items_to_parser(conf_items, p):
+    for item in conf_items:
+        item.add_to(p)
+
+    # Suppress printing of usage synopsis on errors
+    p.error = lambda message: p.exit(2, f'{p.prog}: error: {message}\n')
+
+
+def create_main_parser():
     p = argparse.ArgumentParser(
-        description='Build and sign Unified Kernel Images',
+        description='Create and display information of Unified Kernel Images',
         allow_abbrev=False,
         usage='''\
-ukify [options…] [LINUX INITRD…]
+ukify [options…] [actions]
 ''',
-        epilog='\n  '.join(('config file:', *config_example())),
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
 
-    for item in CONFIG_ITEMS:
-        item.add_to(p)
+    add_items_to_parser(CONFIG_ITEMS_COMMON, p)
 
-    # Suppress printing of usage synopsis on errors
-    p.error = lambda message: p.exit(2, f'{p.prog}: error: {message}\n')
+    return p
+
+
+def create_subparser(subparser, name, conf_items, descr, usage):
```

Comment:
How about
```suggestion
def make_subparser(subparser, name, conf_items, descr, usage):
```
so that we avoid the `create_create_` later